github-issue/Dockerfile

```diff
@@ -2,9 +2,10 @@ FROM python:3.8.3-alpine3.11
 
 WORKDIR /opt/resource/
 
+RUN pip3 install requests python-dateutil
+
 COPY check.py /opt/resource/check
 COPY in.py /opt/resource/in
 COPY out.py /opt/resource/out
 
-RUN pip3 install requests python-dateutil
```

Comment:
moving this up so it is cached when the script files are changed, faster build time in development